vulnerabilities/improvers/__init__.py

```diff
@@ -3,4 +3,4 @@
 
 IMPROVER_REGISTRY = [default.DefaultImprover, importers.nginx.NginxBasicImprover]
 
-improver_mapping = {x.qualified_name(): x for x in IMPROVER_REGISTRY}
+improver_mapping = {x.qualified_name: x for x in IMPROVER_REGISTRY}
```

Comment:
Use the same design as for importers:
```suggestion
IMPROVER_REGISTRY = {x.qualified_name: x for x in IMPROVER_REGISTRY}
```

Reply:
Same as above